test/unittests/evm_other_test.cpp

```diff
@@ -61,6 +61,52 @@ TEST_F(evm_other, evmone_block_max_stack_growth_overflow)
     }
 }
 
+TEST_F(evm_other, evmone_block_gas_cost_overflow_create)
+{
+    // The goal is to build bytecode with as many CREATE instructions (the most expensive one)
+    // as possible but with having balanced stack.
+    // The runtime values of arguments are not important.
+
+    constexpr auto n = 67'097;
+
+    auto code = bytecode{OP_MSIZE};
+    const auto create_pattern = bytecode{} + OP_DUP1 + OP_DUP1 + OP_CREATE;
+    code.reserve(create_pattern.size() * n);
+    for (int i = 0; i < n; ++i)
+        code += create_pattern;
+    EXPECT_EQ(code.size(), 201'292);
```

Comment:
Is this is a new C++ notation for numbers?

Reply:
From C++17 I believe.

Reply:
Of course they couldn't go with the more widely used standard of underscores :wink:

Reply:
I also wander why.